src/python_testing/matter_testing_infrastructure/chip/testing/tasks.py

```diff
@@ -132,6 +132,9 @@ def start(self,
                 raise TimeoutError("Expected output '%r' not found within %s seconds" % (expected_output, timeout))
             self.expected_output = None
 
+    def set_output_match(self, expected_output: str):
```

Comment:
Instead of wrapping "private" method, how about "promoting" it to public (if setting match pattern is required outside of this class)?

Reply:
Acked